geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/pubsub/PubSubIntegrationTest.java

```diff
@@ -16,13 +16,20 @@
 package org.apache.geode.redis.internal.executor.pubsub;
 
 import org.junit.ClassRule;
+import org.junit.contrib.java.lang.system.ProvideSystemProperty;
+import org.junit.rules.RuleChain;
 
 import org.apache.geode.redis.GeodeRedisServerRule;
 
 public class PubSubIntegrationTest extends AbstractPubSubIntegrationTest {
-  @ClassRule
+
   public static GeodeRedisServerRule server = new GeodeRedisServerRule();
 
+  @ClassRule
+  public static RuleChain chain = RuleChain.outerRule(
+      new ProvideSystemProperty("io.netty.eventLoopThreads", "10"))
+      .around(server);
+
```

Comment:
@kirklund would you be happy with the instead of the `RestoreSystemProperties` and the changing of the properties in a `@BeforeClass` block?